File: src/main/java/com/github/graqr/threshr/model/redsky/products/ScheduledDelivery.java
package com.github.graqr.threshr.model.redsky.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.core.annotation.Nullable;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record ScheduledDelivery(
        @JsonProperty("is_active")
        Boolean isActive,
        @Nullable @JsonProperty("location_available_to_promise_quantity")
        Double locationAvailableToPromiseQuantity) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/ScheduledDelivery.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.core.annotation.Nullable
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class ScheduledDelivery(
    @param:JsonProperty("is_active") val isActive: Boolean,
    @param:Nullable @param:JsonProperty("location_available_to_promise_quantity") val locationAvailableToPromiseQuantity: Double
) {}
